lib/utils/colors.js

```diff
@@ -2,132 +2,120 @@ const Settings = {
   enabled: process.env.COLORS !== '0'
 };
 
-function Background() {
-  return this;
-}
+const foregroundColors = {
+  black: '0;30',
+  dark_gray: '1;30',
+  blue: '0;34',
+  light_blue: '1;34',
+  green: '0;32',
+  light_green: '1;32',
+  cyan: '0;36',
+  light_cyan: '1;36',
+  red: '0;31',
+  light_red: '1;31',
+  purple: '0;35',
+  light_purple: '1;35',
+  brown: '0;33',
+  yellow: '1;33',
+  light_gray: '0;37',
+  white: '1;37',
+  stack_trace: '0;90'
+};
 
-function ConsoleColor() {
-  this.background = new Background();
-
-  const foregroundColors = {
-    black: '0;30',
-    dark_gray: '1;30',
-    blue: '0;34',
-    light_blue: '1;34',
-    green: '0;32',
-    light_green: '1;32',
-    cyan: '0;36',
-    light_cyan: '1;36',
-    red: '0;31',
-    light_red: '1;31',
-    purple: '0;35',
-    light_purple: '1;35',
-    brown: '0;33',
-    yellow: '1;33',
-    light_gray: '0;37',
-    white: '1;37',
-    stack_trace: '0;90'
-  };
-
-  const backgroundColors = {
-    black: '40',
-    red: '41',
-    green: '42',
-    yellow: '43',
-    blue: '44',
-    magenta: '45',
-    cyan: '46',
-    light_gray: '47'
-  };
-
-  Object.keys(foregroundColors).forEach(k => {
-    ConsoleColor.prototype[k.toLowerCase()] = (text, background) => {
-      if (!Settings.enabled) {
-        return text;
-      }
+const backgroundColors = {
+  black: '40',
+  red: '41',
+  green: '42',
+  yellow: '43',
+  blue: '44',
+  magenta: '45',
+  cyan: '46',
+  light_gray: '47'
+};
+
+class Background {}
```

Comment:
```suggestion
class Background {
  constructor() {
    return this;
  }
}
```

Reply:
Without a constructor, `new ClassName()` already emits an instance of the class, so we don't need to specify a constructor here. Moreover, even if we specify a constructor, we don't need to explicitly `return this`.

Reply:
Ah! thanks for correcting me. yes, we don't need this